File: DSRFQ.Web/Modules/ServerTypes/Master/VolumeUnitsColumns.ts
import { ColumnsBase, fieldsProxy } from '@serenity-is/corelib';
import { Column } from '@serenity-is/sleekgrid';
import { VolumeUnitsRow } from './VolumeUnitsRow';

export interface VolumeUnitsColumns {
    Id: Column<VolumeUnitsRow>;
    Code: Column<VolumeUnitsRow>;
    Name: Column<VolumeUnitsRow>;
    InsertDate: Column<VolumeUnitsRow>;
    InsertUserId: Column<VolumeUnitsRow>;
    UpdateDate: Column<VolumeUnitsRow>;
    UpdateUserId: Column<VolumeUnitsRow>;
    DeleteDate: Column<VolumeUnitsRow>;
    DeleteUserId: Column<VolumeUnitsRow>;
    IsActive: Column<VolumeUnitsRow>;
}

export class VolumeUnitsColumns extends ColumnsBase<VolumeUnitsRow> {
    static readonly columnsKey = 'Master.VolumeUnits';
    static readonly Fields = fieldsProxy<VolumeUnitsColumns>();
}

File: DSRFQ.Web/Modules/Master/VolumeUnits/VolumeUnitsColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace DSRFQ.Master.Columns;

[ColumnsScript("Master.VolumeUnits")]
[BasedOnRow(typeof(VolumeUnitsRow), CheckNames = true)]
public class VolumeUnitsColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    [EditLink]
    public string Code { get; set; }
    public string Name { get; set; }
    [DateTimeFormatter(DisplayFormat = "yyyy-MM-dd HH:mm"), Width(120)]
    public DateTime InsertDate { get; set; }
    public string InsertBy { get; set; }
    [DateTimeFormatter(DisplayFormat = "yyyy-MM-dd HH:mm"), Width(120)]
    public DateTime UpdateDate { get; set; }
    public string UpdateBy { get; set; }
    [DateTimeFormatter(DisplayFormat = "yyyy-MM-dd HH:mm"), Width(120)]
    public DateTime DeleteDate { get; set; }
    public string DeleteBy { get; set; }
}